StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1131UnitTests.cs

```diff
@@ -0,0 +1,544 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+namespace StyleCop.Analyzers.Test.ReadabilityRules
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Analyzers.ReadabilityRules;
+    using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using TestHelper;
+    using Xunit;
+
+    public class SA1131UnitTests : CodeFixVerifier
+    {
+        [Theory]
+        [InlineData("==", "==")]
+        [InlineData("!=", "!=")]
+        [InlineData(">=", "<=")]
```

Comment:
:exclamation: This operator pairing is incorrect. It should be:

| Old | New |
| --- | --- |
| `==` | `==` |
| `!=` | `!=` |
| `>=` | `<` :exclamation:  |
| `<=` | `>` :exclamation:  |
| `>` | `<=` :exclamation:  |
| `<` | `>=` :exclamation:  |